docs/content_management/field_types/field_type_reference/userfield.md

```diff
@@ -30,11 +30,10 @@ This Field Type validates and stores information about a user.
 
 !!! caution
 
-    Using the MD5-based deprecated hash types is a security risk, because if the hashes are leaked, they are too easily broken by brute-force attacks.
-    The plaintext type offers no security. It was only ever intended for testing, and should never be used now.
+    Old password hash types like MD5 have nummerical hash type value from 1 to 5. These were deprecated in v1.13 and removed in v3.0.
+    Between v1.13 and v3.0 it was possible to update hashes automatically when users logged in.
+    Since v3.0, login is only possible with hash type 6 or larger. Automatic updates of older types on login is not possible anymore.
```

Comment:
```suggestion
    Since v3.0, login is only possible with hash type 6 or larger. Automatic updates of older types on login are not possible anymore.
```